client/analytics/report/orders/config.js

```diff
@@ -27,17 +27,27 @@ export const filters = [
 	{ label: __( 'Advanced Filters', 'wc-admin' ), value: 'advanced' },
 ];
 
+const titleContext =
+	'Sentence fragment, logical, refers to filtering by Order Status, Product, Customer, or ' +
+	'Coupon Code by various criteria. Screen shot for context: ' +
+	'https://cloudup.com/cSsUY9VeCVJ';
+
+const ruleContext =
+	'Sentence fragment, logical, "Is/Is Not/Includes/Excludes" refers to searching for ' +
+	'orders matching a chosen Order Status, Product, Customer, or Coupon Code. ' +
+	'Screen shot for context: https://cloudup.com/cSsUY9VeCVJ';
+
 export const advancedFilterConfig = {
 	status: {
 		labels: {
 			add: __( 'Order Status', 'wc-admin' ),
 			remove: __( 'Remove order status filter', 'wc-admin' ),
 			rule: __( 'Select an order status filter match', 'wc-admin' ),
-			title: __( 'Order Status', 'wc-admin' ),
+			title: _x( 'Order Status', titleContext, 'wc-admin' ),
```

Comment:
I think we should tweak this, and have it be the formatting control with placeholders for the other content (using `interpolateComponents` in the filter itself), which will let translators flip the content as needed (and allow adding the Japanese particles)

```
_x(
	'Order Status {{rule /}} {{filter /}}',
	'Filter for order status as a sentence, see screenshot for context: https://cloudup.com/cSsUY9VeCVJ',
	'wc-admin'
);
```